frontend/src/metabase/query_builder/components/filters/FilterPopover/FilterPopover.styled.tsx

```diff
@@ -20,3 +20,13 @@ export const Button = styled(BaseButton)<Props>`
       alpha(primaryColor, 0.8)};
   }
 `;
+
+export const FilterPopoverSeparator = styled.hr`
+  border: 0;
+  height: 0;
+  border-top: 1px solid ${color("border")};
+`;
+
+export const EmptyFilterPickerPlaceholder = styled.div`
+  margin-bottom: 60px;
```

Comment:
I understand that this style/component exists only because of the `position: absolute` hack on `PopoverFooter`.
I'd rather fix that hack instead of infecting the rest of the code with its repercussions.

The hack can be avoided while preserving [the original goal](https://github.com/metabase/metabase/pull/16308) by adjusting the DOM tree.

Reply:
Yeah, I want to do that as well, but is it worth pursuing for this issue? I think a bunch of tests and things would need to be written to make sure that we haven't messed anything up, and I think it would blow the scope out of this cosmetic P4.

Reply:
If there's no easy way to fix this I'd skip

Reply:
Good points.

How about we just leave a comment next to it explaining its existence and mentioning `PopoverFooter` name?

Reply:
done